container_find_includes.go

```diff
@@ -344,9 +344,7 @@ func findIncludesUntilDone(ctx *types.Context, cache *includeCache, sourceFile t
 			} else {
 				include = IncludesFinderWithRegExp(ctx, string(preproc_stderr))
 				if include == "" {
-					// No include found? Bail out.
-					os.Stderr.Write(preproc_stderr)
-					return i18n.WrapError(preproc_err)
+					ctx.GetLogger().Println(constants.LOG_LEVEL_DEBUG, constants.MSG_FIND_INCLUDES_FAILED, sourcePath)
```

Comment:
I would add here:

    // When no include is found, but gcc exited with an error, something
    // else failed that we cannot fix, so stop processing this file.
    // This intentionally does *not* update the cache, to make sure that
    // on the next run, the error is shown again.
    return nil